contribute/en/Style-and-Tone.md

```diff
@@ -67,9 +67,9 @@ Try not to sound like a robot. Write conversationally, as if you were talking to
 
 Headings use sentence case “This is a heading” unless delineated (eg “Look - This is a heading”).
 
-Nouns use lower case unless they have a formal definition in this style guide (eg “Directory”, “Fader”). The only exception to this is the word “person” or “people” which can remain in lower case.
+Nouns use lower case unless they have a formal definition in this style guide (eg “Directory”, “Fader”) or refer to specific Jamulus features (eg "Audio Alerts", "Jitter Buffer"). The only exception to this is the word “person” or “people” which can remain in lower case.
```

Comment:
OK, the remaining problem here is the sentence:
> The only exception to this is the word “person” or “people” which can remain in lower case.

Does that mean it _must_ or _should_?  Or can you just mix it up?  It's a bit non-directive...

Reply:
Maybe we can just remove that sentence, since “person” or “people” don't have a formal definition for us really.

Reply:
It's under "Standard terms":
> “Person” A human connected to a server (may be on multiple channels). We might say "People on the server", or "People who have muted themselves", rather than Musicians or Channels.

Note "rather than Musicians or Channels", with capitalisation, implying that "Person" and "People" would be.  Of course, we don't...  So it's probably worth adding an explicit mention _there_ about capitalisation, too.

Reply:
Separately:
> We use the following abbreviations:
> Kbit/s; Mbit/s; KByte/s; MByte/s.

I think the lack of clarity people have is whether "K", "M", etc are 1024-based or 1000-based.  It's probably not worth worry about -- and probably worth mentioning that we don't worry about it.